test/cpp/naming/resolver_component_test.cc

```diff
@@ -87,32 +97,50 @@ bool convert_string_to_bool(std::string bool_str) {
   }
 }
 
-vector<GrpcLBAddress> parse_expected_addrs(std::string expected_addrs) {
+void ExpectTokenAndMoveForward(const char *token, std::string &expected_addrs) {
+  size_t next_occurrence = expected_addrs.find(token);
+
```

Comment:
Please eliminate unnecessary blank lines within functions.  In general, I find that if you really need to separate code into logical "chunks", it's better to use a comment line documenting the purpose of the next section than a blank line.

https://google.github.io/styleguide/cppguide.html#Vertical_Whitespace

Reply:
Still need to address this.